src/test/java/org/kainos/ea/team2/api/JobServiceTest.java

```diff
@@ -114,6 +131,131 @@ void whenGetJobsCalled_shouldReturnEmptyList_whenDAOProvidesNoData() throws Fail
         Assertions.assertEquals(jobs.size(), 0);
     }
 
+    /**
+     * Verify that the correct list of band levels is returned
+     * when getBandLevels method is called in service class.
+     * @throws FailedToGetException if sql error caught
+     */
+    @Test
+    void whenGetBandLevelsCalled_jobServicesReturnsListOfBandLevels() throws FailedToGetException, SQLException {
+
+        // create band levels
+        BandLevel bandLevel1 = new BandLevel(1, "Trainee");
+        BandLevel bandLevel2 = new BandLevel(2, "Associate");
+
+        // create array list of band levels
+        List<BandLevel> testBandLevels = new ArrayList<>();
+        // add band levels to array list
+        testBandLevels.add(bandLevel1);
+        testBandLevels.add(bandLevel2);
+
+        // specify return of band levels when dao is called
+        Mockito.when(jobDao.getBandLevels()).thenReturn(testBandLevels);
+
+        // job service class calls to job dao class to get band levels
+        List<BandLevel> bandLevels = jobService.getBandLevels();
+
+        // check band levels returned from job service class equal test band levels list
+        Assertions.assertEquals(bandLevels,testBandLevels);
```

Comment:
Small detail, these arguments should be the other way around (expected result first, then the actual result)